### ISSUE TITLE
Android Tensorflow Yolo

### ISSUE THREAD

**Issue by xinyee1997** [AlexeyAB/darknet#5073]
Have anyone of you successfully built yolo on android tensorflow. I am not getting any result when using yolo in android tensorflow. 

anyone know the result ?

**Comment by AlexeyAB** [comment]
Did you use this manual? https://docs.openvinotoolkit.org/latest/_docs_MO_DG_prepare_model_convert_model_Convert_Model_From_TensorFlow.html#converting-a-darknet-yolo-model

**Comment by xinyee1997** [comment]
@AlexeyAB  Nope. I follow this manual http://seangtkelley.me/blog/2017/12/23/using-custom-yolov2-model-on-android

I will spend time on the manual u gave me and see how's the result . Thank you very much :)
